edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
+
+    img {
+      height: 160px;
+      max-width: 264px;
     }
   }
+
+  .course-info .container .course-sidebar {
+    width: 100%;
+  }
+}
+
+.wrapper-course-material .course-tabs {
+  padding: 0;
+}
+
+.tabs .tab a, .tabs .tab a:visited {
+  padding: 10px 15px 7px 15px;
+}
+
+body.view-in-course .wrapper-course-material .course-material .course-tabs {
+  padding: 0;
 }
 
-// +Search Input
-// ====================
-.find-courses {
-  .wrapper-search-context {
-    @include outer-container;
+div.wrapper-course-material .course-tabs .tab a,
+div.wrapper-course-material .course-tabs .tab a:visited {
+  padding: 10px 15px 7px 15px;
+}
 
-    @include rtl() { $layout-direction: "RTL"; }
+.course-wrapper section#course-content #main {
+  max-width: 980px;
+  display: block;
+  margin: 0 auto;
+}
 
-    .search-status-label {
-      @extend %t-title3;
+div.course-wrapper .course-content .xblock.xblock-student_view-vertical,
+div.course-wrapper .courseware-results-wrapper .xblock.xblock-student_view-vertical {
+  max-width: 100%;
+}
 
-      @include span-columns(9);
+//info
+section.updates div .updates-article {
+  background: $background-color-info;
+}
 
-      min-height: $course-search-input-height;
+.course-outline .block-tree .section ol.outline-item .subsection a.outline-item:hover,
+.course-outline .block-tree .section ol.outline-item .subsection a.outline-item:focus {
+  background-color: $primary;
+  color: $inverse;
+}
 
-      @include media($bp-tiny) {
-        @include fill-parent();
-        @include font-size(20);
-      }
 
-      @include media($bp-small) {
-        @include span-columns(4);
-        @include font-size(20);
-      }
+div.info-wrapper section.updates>ol .updates-article,
+div.info-wrapper section.updates section .updates-article,
+div.info-wrapper section.updates div article.updates-article {
+  background-color: $background-color-info;
+}
 
-      @include media($bp-medium) {
-        @include span-columns(4);
-      }
+.wrapper-course-material .course-tabs li.tab a,
+.wrapper-course-material .course-tabs li.tab a:visited {
+  color: $primary-font-color;
+}
 
-      @include media($bp-large) {
-        @include span-columns(8);
-      }
-    }
+.xmodule_display.xmodule_SequenceModule
+.sequence-nav ol li button div.sequence-tooltip {
+  color: $primary-font-color;
+}
 
-    .wrapper-search-input {
-      @include span-columns(3);
+body.view-in-course div.info-wrapper section.handouts a {
+  color: $primary;
+}
 
-      @extend %ui-depth0;
+div.info-wrapper section.updates div article .toggle-visibility-element {
+  color: $primary;
+}
 
-      position: relative;
+//course-course
+.xmodule_display.xmodule_SequenceModule div.sequence-nav ol li button span.icon {
+  font-size: 100%;
+  color: $pearson-dark-blue-courseware;
+}
 
-      @include media($bp-tiny) {
-        @include fill-parent();
-      }
+h2.hd.hd-2.unit-title,
+.xblock .xblock h2,  {
+  font-size: 2.25em;
+  font-family: Playfair Display, serif;
```

Comment:
Is there already a variable for this? @luismorenolopera

Reply:
yes